package.json

```diff
@@ -50,5 +49,9 @@
     "tracking",
     "raygun",
     "clientside"
-  ]
+  ],
+  "dependencies": {
+    "@wdio/codemod": "^0.12.0",
+    "jscodeshift": "^0.15.0"
```

Comment:
all dependencies for the JS provider are going to be developer deps as these are used to build the project

Reply:
Cleaned up